model/model.py

```diff
@@ -0,0 +1,109 @@
+import os
+import time
+import json
+import argparse
+import duckdb
```

Comment:
would be good to update the requirements.txt if there are new ones (I think duckdb is new?)

Reply:
I've updated the requirements.txt, though the requirements.txt now is still broken (specifying --no-deps will cause certain library installations to fail (e.g. scikit-learn which depends on NumPy), however removing it will introduce dependency conflicts). I'm also just wondering if the requirements contain a lot more packages than we actually need (maybe as a result of pip3 freeze)? I tried running pipreqs, but I'm on Python 3.8 instead 3.6.

Reply:
Let's clean this up in the next iteration